com.unity.netcode.gameobjects/Editor/NetworkManagerEditor.cs

```diff
@@ -356,5 +363,80 @@ public override void OnInspectorGUI()
                 }
             }
         }
+
+        void DrawMultiplayerToolsInfoTip()
+        {
+            const string dismissedKey = "NGO_Tools_Tip_Dismissed";
```

Comment:
nit: I worry just a little about the potential for this string to collide with another key. Maybe it could be further distinguished from other NGO purposes by calling it `"NGO_Multiplayer_Tools_Prompt_Dismissed"` or something like that? Right now it's just one character away from `"NGO_Tool_Tip_Dismissed"`

Reply:
Hm, seems like a small chance but simple enough to avoid. How about
`NGO_Tip_InstallMPTools_Dismissed`

Reply:
Sounds good!

Reply:
also nit: we don't use NGO acronym in the SDK. it's either Netcode for GameObjects or Netcode. in this case, I believe it should be `"Netcode_Tools_Tip_Dismissed"` or maybe just directly `"Multiplayer_Tools_Tip_Dismissed"`

another nit: shouldn't this whole method also be wrapped into `#if !MULTIPLAYER_TOOLS`?

Reply:
It could be, but doesn't need to be. It's better to have less code wrapped in conditional directives wherever possible
I'll update the key to Netcode